File: in-motion-server/User/IMS.User.Models/Dto/Outgoing/UpdatedUserProfileVideoDto.cs
namespace IMS.User.Models.Dto.Outgoing;

public class UpdatedUserProfileVideoDto
{
    public string Id { get; set; }
    public string UserId { get; set; }
    public string Filename { get; set; }
    public string BucketName { get; set; }
    public string BucketLocation { get; set; }
    public string ContentType { get; set; }
}

File: in-motion-ios/inMotion/inMotion/Models/DTO/User/Incoming/UpdatedUserProfileVideoDto.swift
//
//  UpdatedUserProfileVideoDto.swift
//  inMotion
//
//  Created by Kamil Pietrak on 31/10/2023.
//

import Foundation

struct UpdatedUserProfileVideoDto: Codable {
    public var id: UUID
    public var userId: UUID
    public var filename: String
    public var bucketName: String
    public var bucketLocation: String
    public var contentType: String
}
